zapcore/encoder.go

```diff
@@ -168,6 +168,12 @@ func NanosDurationEncoder(d time.Duration, enc PrimitiveArrayEncoder) {
 	enc.AppendInt64(int64(d))
 }
 
+// MilliSecondsDurationEncoder serializes a time.Duration to an integer number of
+// milliseconds elapsed.
+func MilliSecondsDurationEncoder(d time.Duration, enc PrimitiveArrayEncoder) {
```

Comment:
Should this be called `MillisDurationEncoder` to match the naming of the other methods

Reply:
Following up on this, to match `NanosDurationEncoder`, this should be `MillisDurationEncoder`

Reply:
> NanosDurationEncoder

Done